packages/sdk-webpack/src/config/webpack.config.js

```diff
@@ -391,12 +391,14 @@ module.exports = function (webpackEnv) {
                             options: {
                                 // customize: require.resolve('babel-preset-react-app/webpack-overrides'),
                                 presets: [
-                                    // [
-                                    //     require.resolve('babel-preset-react-app'),
-                                    //     {
-                                    //         runtime: hasJsxRuntime ? 'automatic' : 'classic',
-                                    //     },
-                                    // ],
+                                    [
```

Comment:
@mihaiblaga89 can you wrap this with RNV OVERRIDE comments to ensure these changes don't get overridden during future upgrades